pages/common/nm.md

```diff
@@ -0,0 +1,19 @@
+# nm
+
+> List symbol names in object files
+
+- List global (extern) functions in a file (prefixed with T)
+
+`nm -g {{file.o}}`
+
+- Demangle C++ symbols (make them readable)
+
+`nm {{file.o}} | c++filt`
```

Comment:
We do have a linux folder and are considering one for bsd (I suppose it would be ok if you created one yourself, /cc @rprieto), so the distinction could be done in that way instead of using the lowest common denominator in a common/ page.